rita/src/rita_common/tunnel_manager/mod.rs

```diff
@@ -78,6 +83,24 @@ impl SystemService for TunnelManager {
             self.listen_interfaces.insert(i);
         }
         trace!("Loaded listen interfaces {:?}", self.listen_interfaces);
+
+        #[cfg(not(test))]
+        {
+            Arbiter::registry().init_actor(|_| {
+                //TODO: make the configurable when trust-dns-resolver serde issue is solved
+                //default is 8.8.8.8
+                Connector::new(ResolverConfig::default(), ResolverOpts::default())
```

Comment:
have you tried setting dnssec to true?

Reply:
not yet, I think we can try that when can change the settings via config file instead of messing with the code, but it _does_ say dnssec validation is implemented (FWIW)